master/docs/tutorial/tour.rst

```diff
@@ -166,13 +166,14 @@ First, start an IRC client of your choice, connect to irc.freenode.org and join
 In this example we will use #buildbot-test, so go join that channel.
 (*Note: please do not join the main buildbot channel!*)
 
-Edit the config and look for the *STATUS TARGETS* section.
-Enter these lines below the WebStatus line in :file:`master.cfg`::
+Edit :file:'master.cfg' and look for the *STATUS TARGETS* section. At the
+end of that section, just below the WebStatus line::
 
```

Comment:
You're correct! I now see that sample.cfg was changed here and that code was removed from sample.cfg. I was looking at the released version. I can reword but I wanted to test but I'm having trouble installing. I'll ask in the devel list.